README.md

```diff
@@ -3164,6 +3164,15 @@ Sets the [SSLProxyMachineCertificateFile](http://httpd.apache.org/docs/current/m
     }
 ~~~
 
+##### `ssl_proxy_check_peer_cn`
```

Comment:
Can you move this above the section `ssl_proxy_machine_cert` as to keep the params in alphabetical order?